components/Tabs2/src/MDCTabBarView.m

```diff
@@ -16,4 +16,49 @@
 
 @implementation MDCTabBarView
 
+#pragma mark - Initialization
+
+- (instancetype)init {
+  self = [super init];
+  if (self) {
+    self.items = @[];
+  }
+  return self;
+}
+
+#pragma mark - Properties
+
+- (void)setItems:(NSArray<UITabBarItem *> *)items {
+  NSParameterAssert(items);
+
+  if (_items != items && ![_items isEqual:items]) {
```

Comment:
Please access other properties via the appropriate setters/getters here.  The exception is the actual method you're in (setItems).

Reply:
Fix applied.